app/models/claims/state_machine.rb

```diff
@@ -79,7 +79,7 @@ def self.included(klass)
         before_transition on: %i[reject refuse], do: :set_amount_assessed_zero!
 
         around_transition any => NON_VALIDATION_STATES.map(&:to_sym) do |claim, transition, block|
-          validation_state = %i[authorise part_authorise].include?(transition.event) ? :only_amount_assessed : :all
+          validation_state = %i[authorise authorise_part].include?(transition.event) ? :only_amount_assessed : :all
```

Comment:
Yikes.. good catch!

Reply:
indeed, it means that part authorise may not have required an assessment in the past...doh!